### ISSUE TITLE
Hi, could you please provide the code and related files for testing your best model on the test set?

### ISSUE THREAD

**Issue by GarinLL** [Chaos96/fourierft#15]
@Chaos96

**Comment by zqgao22** [comment]
> _No description provided._

Hi, following most of the PEFT papers (e.g. LoRA, VeRA, AdaLoRA...), we report the best results on the validation set. Due to the fact that the labels of GLUE's test sets are not all available, the original RoBERTa paper (https://arxiv.org/pdf/1907.11692) applies two settings. The first one is the one we use. Moreover, I think we can also (a) apply the standard train-val-test paradigm and submit each test result to the GLUE leaderboard, or (b) split a validation set from the training set and regard the native validation set as the test set.

Best.

**Comment by GarinLL** [comment]
I see, your result is the median of the best validation metric from 5 training runs. But what do the subscripts on the numbers in the graph mean?
![image](https://github.com/user-attachments/assets/b731f7e2-ebd8-4661-8a29-05504a7c7b5b)
